nidirect_webforms/templates/nidirect-webforms-quiz-answer-feedback.html.twig

```diff
@@ -0,0 +1,9 @@
+{% set classes = [
+  passed ? 'success-notice' : 'error-notice'
+]
+%}
+<div {{ attributes.addClass(classes) }}>
+  <h2>{{ title }}</h2>
+  <p class="question">{{ question }}</p>
+  <div class="feedback"> {{ feedback }}</div>
```

Comment:
don't think there should be a space before {{ feedback }} ?

Reply:
Good spot!